### PR TITLE
Using new RestBridge API

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

File: apps/livreGUI/stimuliController/StimuliController.cpp [hunk 1]
```diff
@@ -245,6 +263,9 @@ StimuliController::StimuliController( Controller& controller,
 
     setWindowFlags(( windowFlags() ^ Qt::Dialog) | Qt::Window );
 
+    connect( _ui.btnPlayPauseSimulation, SIGNAL(clicked()),
```
Comment (bilgili) [inline apps/livreGUI/stimuliController/StimuliController.cpp]: ResumePause simulation

File: livre/eq/Client.cpp [hunk 2]
```diff
@@ -120,21 +120,21 @@ int Client::run()
...
     frameData.getVolumeSettings()->setURI( _applicationParameters.dataFileName);
 
     // 3. init config
     lunchbox::Clock clock;
-    if( !config->init())
+    if( !config->init( argc, argv ))
     {
         server->releaseConfig( config );
         disconnectServer( server );
+        exitLocal();
```
Comment (bilgili) [inline livre/eq/Client.cpp]: The equalizer changes are because ?

File: livre/eq/zeq/communicator.cpp [hunk 3]
```diff
@@ -341,7 +317,7 @@ class Communicator::Impl
     {
         SubscriberPtr subscriber(
                     new ::zeq::Subscriber(
-                        servus::URI( _config.getApplicationParameters().zeqSchema )));
+                        ::zeq::URI( _config.getApplicationParameters().zeqSchema )));
```
Comment (bilgili) [inline livre/eq/zeq/communicator.cpp]: we moved URI to zeq ?

File: CMakeLists.txt [hunk 4]
```diff
@@ -29,7 +29,7 @@ set(COMMON_PROJECT_DOMAIN ch.epfl.bluebrain)
 
 include(Common)
 if(NOT MSVC) # cppnetlib does not build on Windows
-  git_subproject(RESTBridge https://github.com/BlueBrain/RESTBridge.git 4f16cfb)
+  git_subproject(RESTBridge https://github.com/BlueBrain/RESTBridge.git summit)
```
Comment (bilgili) [inline CMakeLists.txt]: Having this (summit) in master branch seems wrong to me. We are doing temporary hardcodings for summit.

Comment (eile) [inline CMakeLists.txt]: eile  wants to merge 8 commits into BlueBrain:summit from eile:summit

**Comment by eile** [comment]
I'm still preparing this PR.

File: CMakeLists.txt (hunk 4 above)
Comment (bilgili) [inline CMakeLists.txt]: ouch.

**Pull request closed without merging**